CollapseLauncher/XAMLs/MainApp/Pages/SettingsPage.xaml.cs

```diff
@@ -788,16 +791,42 @@ private bool IsIncludeGameLogs
 
         private bool IsShowRegionChangeWarning
         {
-            get => LauncherConfig.IsShowRegionChangeWarning;
+            get
+            { 
+                var value = LauncherConfig.IsShowRegionChangeWarning;
+
+                ToggleChangeRegionInstant.Visibility = !value ? Visibility.Visible : Visibility.Collapsed;
+                return value;
+            }
             set
             {
                 IsChangeRegionWarningNeedRestart = true;
```

Comment:
hide the warning if we switch it back

Reply:
Fixed in https://github.com/CollapseLauncher/Collapse/pull/449/commits/dd2901c974e260044f78024631e13d69988b9e12